### PR TITLE
chore(config): add dependabot config

### LINKED ISSUE #23: Setup dependabot
Would be good to have dependencies up to date

### PR DESCRIPTION
**Pull request by skjnldsv**
close #23

Signed-off-by: John Molakvoæ <skjnldsv@users.noreply.github.com>

### CODE REVIEW

File: .github/dependabot.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+# package ecosystems to update and where the package manifests are located.
+# Please see the documentation for all configuration options:
+# https://help.github.com/github/administering-a-repository/configuration-options-for-dependency-updates
+version: 2
+updates:
+  - package-ecosystem: npm
+    directory: "/"
+    schedule:
+      interval: weekly
+      day: saturday
```
Comment (ChristophWurst) [inline .github/dependabot.yml:11]: nit: make it a weekday so my notification inbox isn't flooded monday morning

**Review by ChristophWurst — APPROVED** [review]
I will grant you bonus points for conventional commit messages 

```yaml
  commit-message:
    prefix: chore
    include: scope
```

File: .github/dependabot.yml (hunk 1 above)
Comment (skjnldsv) [reply .github/dependabot.yml:11]: But then CI is filled
If you're reactive to merge things, putting it to daily makes it smoother. But you have to merge things and not go stale :shrug: https://github.com/nextcloud/nextcloud-cypress/blob/master/.github/dependabot.yml

Comment (ChristophWurst) [reply .github/dependabot.yml:11]: At night it's filled as well?

You can leave it at Saturday if you prefer that.

**Pull request merged**